url7a69e77641_benmouh_MySgbd_tgz-pJ8-Demo_CheckRowNumJ8/src/org/sgbd/Mysgbd/parser/MysgbdParserTokenManager.java

```diff
@@ -0,0 +1,1654 @@
+            return jjStartNfaWithStates_0(2, 6, 48);
+         return jjMoveStringLiteralDfa3_0(active0, 0x10080000000000L, active1, 0x10L);
+      case 69:
+      case 101:
+         return jjMoveStringLiteralDfa3_0(active0, 0L, active1, 0x100L);
+      case 71:
+      case 103:
+         if ((active0 & 0x400L) != 0L)
+            return jjStartNfaWithStates_0(2, 10, 48);
+         break;
+      case 73:
+      case 105:
+         return jjMoveStringLiteralDfa3_0(active0, 0x60000006000000L, active1, 0x808L);
+      case 75:
+      case 107:
+         return jjMoveStringLiteralDfa3_0(active0, 0x4000000000L, active1, 0L);
+      case 76:
+      case 108:
+         if ((active0 & 0x20L) != 0L)
+            return jjStartNfaWithStates_0(2, 5, 48);
+         return jjMoveStringLiteralDfa3_0(active0, 0xa04800000200000L, active1, 0x20L);
+      case 77:
+      case 109:
+         if ((active1 & 0x1L) != 0L)
+            return jjStartNfaWithStates_0(2, 64, 48);
+         return jjMoveStringLiteralDfa3_0(active0, 0x1000000030000L, active1, 0L);
+      case 78:
+      case 110:
+         if ((active0 & 0x20000000000L) != 0L)
+         {
+            jjmatchedKind = 41;
+            jjmatchedPos = 2;
+         }
+         return jjMoveStringLiteralDfa3_0(active0, 0x40000041000L, active1, 0L);
+      case 79:
+      case 111:
+         return jjMoveStringLiteralDfa3_0(active0, 0x68002000L, active1, 0L);
+      case 82:
+      case 114:
+         if ((active0 & 0x10000000L) != 0L)
+            return jjStartNfaWithStates_0(2, 28, 48);
+         return jjMoveStringLiteralDfa3_0(active0, 0L, active1, 0x4c0L);
+      case 83:
+      case 115:
+         return jjMoveStringLiteralDfa3_0(active0, 0x200c00000L, active1, 0L);
+      case 84:
+      case 116:
+         if ((active0 & 0x200000000000L) != 0L)
+            return jjStartNfaWithStates_0(2, 45, 48);
+         else if ((active0 & 0x1000000000000000L) != 0L)
+            return jjStartNfaWithStates_0(2, 60, 48);
+         return jjMoveStringLiteralDfa3_0(active0, 0x101c00100800L, active1, 0x200L);
+      case 85:
+      case 117:
+         return jjMoveStringLiteralDfa3_0(active0, 0x80000L, active1, 0L);
+      case 86:
+      case 118:
+         return jjMoveStringLiteralDfa3_0(active0, 0x80000000L, active1, 0L);
+      case 87:
+      case 119:
+         if ((active0 & 0x400000000000000L) != 0L)
+            return jjStartNfaWithStates_0(2, 58, 48);
+         return jjMoveStringLiteralDfa3_0(active0, 0x400000000000L, active1, 0L);
+      case 88:
+      case 120:
+         if ((active0 & 0x10000000000L) != 0L)
+            return jjStartNfaWithStates_0(2, 40, 48);
+         break;
+      case 89:
+      case 121:
+         if ((active0 & 0x80L) != 0L)
+            return jjStartNfaWithStates_0(2, 7, 48);
+         break;
+      default :
+         break;
+   }
+   return jjStartNfa_0(1, active0, active1);
+}
+static private int jjMoveStringLiteralDfa3_0(long old0, long active0, long old1, long active1)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This method contains 97 lines of code, not including blank lines or lines with only comments, Java punctuation characters, identifiers, or literals. By comparison, 99% of the methods in the CodeGuru reference dataset contain fewer lines of code. Large methods might be difficult to read and have logic that is hard to understand and test. We recommend that you simplify this method or break it into multiple methods.